modules/openapi-generator/src/main/resources/Java/pojo.mustache

```diff
@@ -129,6 +129,7 @@ public class {{classname}} {{#parent}}extends {{{parent}}} {{/parent}}{{#parcela
   {{/maximum}}
    * @return {{name}}
   **/
+{{^required}}  @javax.annotation.Nullable {{/required}}
```

Comment:
@kassim thanks for the PR. Can you use the following instead to avoid blank lines or trailing spaces?
```
  {{^required}}
  @javax.annotation.Nullable
  {{/required}}
```

Reply:
I'll do it in another PR.

Reply:
Filed https://github.com/OpenAPITools/openapi-generator/pull/3453